docs/Early Access - Using Global Profile Configuration.md

```diff
@@ -53,6 +53,19 @@ The global profile functionality simplifies profile management by enabling you t
 
 **Note**: A project context takes precedence over global configuration.
 
+### Create a Team Configuration File
+
+Create a team configuration file.
+
+1. Navigate to the explorer tree.
+2. Hover over **DATA SETS**, **USS**, or **JOBS**.
+3. Click the **+** icon.
+4. Select **Create a New Team Configuration File**.
+5. Follow the instructions to complete the team configuration profile creation.
+6. Reload your VS Code window when prompted.
+
+Your team configuration file appears in your `.zowe` folder.
+
```

Comment:
this should state that it will either show up in the `.zowe` directory or in the `workspace` directory depending on the users choice of config file.

Reply:
Added